src/lib/resource.js

```diff
@@ -386,7 +386,7 @@ const getResourceMetaDataJob = async (job) => {
 
   try {
     // Determine if this is an ECLKC resource.
-    const isEclkc = resourceUrl.includes('eclkc.ohs.acf.hhs.gov');
```

Comment:
We need to still do this for ECLKC as there will be some time where we get both, Please add not replace the domain for additional processing.

Reply:
Won't they be phasing out the old URL.

Reply:
I guess another concern is after they phase it out this code might fail if it tries to pull metadata for a url that no longer works.

Reply:
yes, but it will still be in some people list of saved resources, so we will still be getting them for a while.

Reply:
done